server/base/src/main/java/org/apache/accumulo/server/conf/ServerConfigurationFactory.java

```diff
@@ -87,13 +88,15 @@ static void expireAllTableObservers() {
     }
   }
 
-  private final Instance instance;
+  private final ServerInfo info;
+  private final ClientContext context;
   private final String instanceID;
   private ZooCacheFactory zcf = new ZooCacheFactory();
 
-  public ServerConfigurationFactory(Instance instance) {
-    this.instance = instance;
-    instanceID = instance.getInstanceID();
+  public ServerConfigurationFactory(ServerInfo info) {
+    this.info = info;
+    context = new ClientContext(info);
```

Comment:
Would it make more sense to have something like `info.getClientContext()`?  This could result in a single context per ServerInfo, which may be good.

Reply:
This only takes ServerContext now.